src/plugins/kibana/public/dashboard/index.js

```diff
@@ -108,7 +107,24 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
         options: require('plugins/kibana/dashboard/partials/options.html'),
         filter: require('ui/chrome/config/filter.html'),
         interval: require('ui/chrome/config/interval.html')
-      });
+      };
+      $scope.topNavMenu = [{
```

Comment:
it seems like the property must be called this right now, this should be passed somewhere in the view. Ditto for all the other views.